src/syft/lib/torch/allowlist.py

```diff
@@ -667,89 +667,90 @@
-# ] = "torch.utils.data.dataloader._SingleProcessDataLoaderIter"
+allowlist["torch.utils.data.DataLoader"] = "torch.utils.data.DataLoader"
+allowlist[
+    "torch.utils.data.DataLoader.__iter__"
+] = "torch.utils.data.dataloader._SingleProcessDataLoaderIter"
+allowlist["torch.utils.data.DataLoader.__len__"] = "syft.lib.python.Int"
+allowlist[
+    "torch.utils.data.dataloader._SingleProcessDataLoaderIter"
+] = "torch.utils.data.dataloader._SingleProcessDataLoaderIter"
+allowlist[
+    "torch.utils.data.dataloader._SingleProcessDataLoaderIter.__len__"
+] = "syft.lib.python.Int"
+allowlist[
+    "torch.utils.data.dataloader._SingleProcessDataLoaderIter.__iter__"
+] = "torch.utils.data.dataloader._SingleProcessDataLoaderIter"
 
 # working for part
-# allowlist[
-#     "torch.utils.data.dataloader._SingleProcessDataLoaderIter.__next__"
-# ] = "torch.Tensor"
-# allowlist[
-#     "torch.utils.data.dataloader._SingleProcessDataLoaderIter.next"
-# ] = "torch.Tensor"
+allowlist[
+    "torch.utils.data.dataloader._SingleProcessDataLoaderIter.__next__"
+] = "torch.Tensor"
+allowlist[
+    "torch.utils.data.dataloader._SingleProcessDataLoaderIter.next"
+] = "torch.Tensor"
 
 # we are returning syft.lib.python.List so that we can __getitem__ on the return of
 # enumerate(train_loader)
-# allowlist[
-#     "torch.utils.data.dataloader._SingleProcessDataLoaderIter.__next__"
-# ] = "syft.lib.python.List"
-# allowlist[
-#     "torch.utils.data.dataloader._SingleProcessDataLoaderIter.next"
-# ] = "syft.lib.python.List"
+allowlist[
+    "torch.utils.data.dataloader._SingleProcessDataLoaderIter.__next__"
+] = "syft.lib.python.List"
+allowlist[
+    "torch.utils.data.dataloader._SingleProcessDataLoaderIter.next"
+] = "syft.lib.python.List"
 
 # Functional
-# allowlist["torch.nn.functional.relu"] = "torch.Tensor"
-# allowlist["torch.nn.functional.max_pool2d"] = "torch.Tensor"
-# allowlist["torch.nn.functional.log_softmax"] = "torch.Tensor"
-# allowlist["torch.nn.functional.nll_loss"] = "torch.Tensor"
-# allowlist["torch.flatten"] = "torch.Tensor"
+allowlist["torch.nn.functional.relu"] = "torch.Tensor"
+allowlist["torch.nn.functional.max_pool2d"] = "torch.Tensor"
+allowlist["torch.nn.functional.log_softmax"] = "torch.Tensor"
+allowlist["torch.nn.functional.nll_loss"] = "torch.Tensor"
+allowlist["torch.flatten"] = "torch.Tensor"
 
 # Optimizers
-# allowlist["torch.optim.Adadelta"] = "torch.optim.Adadelta"
-# allowlist["torch.optim.lr_scheduler.StepLR"] = "torch.optim.lr_scheduler.StepLR"
-# allowlist["torch.optim.lr_scheduler.StepLR.step"] = "syft.lib.python.SyNone"
-# allowlist["torch.optim.Adadelta.zero_grad"] = "syft.lib.python.SyNone"
-# allowlist["torch.optim.Adadelta.step"] = "syft.lib.python.SyNone"
+allowlist["torch.optim.Adadelta"] = "torch.optim.Adadelta"
+allowlist["torch.optim.lr_scheduler.StepLR"] = "torch.optim.lr_scheduler.StepLR"
+allowlist["torch.optim.lr_scheduler.StepLR.step"] = "syft.lib.python._SyNone"
```

Comment:
Q: What is ```_SyNone```?

Reply:
That is the renamed Constructor Class for SyNone which is the shared instance variable.